README.md

```diff
@@ -67,6 +67,48 @@ Complete the following steps before you deploy the Secure Landing Zone module.
 
     Select **All Identity and Access enabled services** when you assign access to the group.
 
+### Steps to prevent Destroy and recreation of cluster.
```

Comment:
If in it's own file, the title would probably be something like
`Updating to another version of the Red Hat OpenShift pattern`